cli/cli.go

```diff
@@ -94,6 +94,70 @@ func main() {
 				},
 			},
 		},
+		{
+			Name: "delete",
+			Subcommands: []cli.Command{
+				{
+					Name:      "pipeline",
+					Usage:     "delete pipeline",
+					ArgsUsage: "<pipeline-name>",
+					Before:    requireNameArg,
+					Action:    deletePipeline,
+				},
+				{
+					Name:      "build",
+					Usage:     "delete build",
+					ArgsUsage: "<pipeline-name>",
+					Flags: []cli.Flag{
+						cli.IntFlag{
+							Name:  "build, b",
+							Usage: "build number, if not provided will not proceed deletion",
+						},
+					},
+					Before: requireNameArg,
+					Action: deleteBuild,
+				},
+			},
+		},
+		{
+			Name:  "deploy",
+			Usage: "deploy kontinuous app in the cluster",
+			Subcommands: []cli.Command{
+				{
+					Name:   "remove",
+					Usage:  "remove kontinuous app in the cluster",
+					Action: removeDeployedApp,
+				},
+			},
+			Flags: []cli.Flag{
+				cli.StringFlag{
+					Name:  "namespace",
+					Usage: "Required, kontinuous namespace",
+				},
+				cli.StringFlag{
+					Name:  "accesskey",
+					Usage: "Required, s3 access key",
+				},
+				cli.StringFlag{
+					Name:  "secretkey",
+					Usage: "Required, s3 secret key",
+				},
+				cli.StringFlag{
+					Name:  "authcode",
+					Usage: "Required, jwt authorization code",
+				},
+				cli.StringFlag{
+					Name:  "clientid",
+					Usage: "Required, github client id",
```

Comment:
When manually deploying, it is optional if we use Auth0. For the deploy though, since we're deploying kontinuous-ui, it becomes required for github integration. May need to make it clearer in the README.